runtime/rasdump/javadump.cpp

```diff
@@ -2162,11 +2165,28 @@ JavaCoreDumpWriter::writeUnmountedThreads(void)
 	PORT_ACCESS_FROM_JAVAVM(_VirtualMachine);
 
 	_OutputStream.writeCharacters(
-			"1XMVTHDINFO    Unmounted Threads\n"
-			"NULL           =================\n"
+			"1XMVTHDINFO    Unmounted Virtual Threads\n"
+			"NULL           =========================\n"
 			"NULL\n"
 		);
 
+	U_32 vthreadCounts[2] = {0, 0};
+	_VirtualMachine->memoryManagerFunctions->j9mm_iterate_all_continuation_objects(_VirtualMachine->mainThread, PORTLIB, 0, vthreadCountIteratorCallback, vthreadCounts);
+
+	_OutputStream.writeCharacters("1XMVPOOLINFO    JVM Virtual Thread info:");
+	_OutputStream.writeCharacters("\n2XMVPOOLTOTAL       Current total number of virtual threads:         ");
+	_OutputStream.writeInteger((vthreadCounts[0] + vthreadCounts[1]), "%i");
+	_OutputStream.writeCharacters("\n2XMVPOOLNEW         Current total number of new virtual threads:     ");
+	_OutputStream.writeInteger(vthreadCounts[0], "%i");
+	_OutputStream.writeCharacters("\n2XMVPOOLSTARTED     Current total number of started virtual threads: ");
```

Comment:
Why do these names use `POOL`?

Reply:
I just copied the format from `JavaCoreDumpWriter::writeThreadSection(void)`

Reply:
I don't think using `POOL` in these keys is appropriate.